5 - Shor's Algorithm on logical cat qubits using the Qrisp framework.ipynb

```diff
@@ -18,43 +18,42 @@
     "\n",
-    "Shor's algorithm requires considerably low error rates to give proper results - we will talk about numbers in a following section, when we will do a sketch of the resource estimation of the algorithm - and therefore error correction is needed. So we tought - wait a second, why don't we use cat qubits then, which are the qubits of the future?\n",
+    "Shor's algorithm requires considerably low error rates to give proper results - we will talk about numbers in a following section, when we will do a sketch of the resource estimation of the algorithm - and therefore error correction is needed. Cat qubits are best suited for this goal for their outstanding characteristics so let's simulate their behaviour to solve the problem.\n",
     "\n",
     "### Why Qrisp?\n",
     "\n",
     "Qrisp is a high-level programming language for creating and compiling quantum algorithms which features numerous characteristics to smooth the workflow when writing and executing a quantum algorithm.\n",
     "If you want to get familiar with the Qrisp framework, have a look at the dedicated [series of tutorials](https://qrisp.eu/general/tutorial/index.html) available on Qrisp's website.\n",
     "\n",
-    "Qrisp provides compatibility with Qiskit's framework and it is possible to run Qrisp code directly on the Qiskit's backend making use of the `VirtualQiskitBackend`. This allows us to instantiate the emulator using the [Alice & Bob Qiskit Provider]() as backend and run the code on a cat qubits architechture.\n"
+    "Qrisp provides compatibility with Qiskit's framework and it is possible to run Qrisp code directly on the Qiskit's backend making use of the `VirtualQiskitBackend`. This allows us to instantiate the emulator using the [Alice & Bob Qiskit Provider](https://alice-bob.notion.site/Alice-Bob-Felis-documentation-52e554ccbea54e34831761f083883e49?pvs=4) as backend and run the code on a cat qubits architechture."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
-   "id": "3a32e67d-f628-4f47-a945-b44a342cd6ac",
+   "execution_count": 1,
```

Comment:
Can you use the "clear all outputs" option for the commit not to have your previous run data?